User/v jandrew/revised enterprise policy (#2337)
MicrosoftDocs/edge-developer: microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

We are aware that Chromium has revised the timelines for Manifest V2 sunset. We will independently decide on MV3 migration timelines for Microsoft Edge Add-ons and share an update here. We continue to analyze the concerns raised by the extension developers and explore the optimal path for Microsoft Edge Add-ons ecosystem. Meanwhile, please refer to the [Chromium timelines](https://developer.chrome.com/docs/extensions/mv3/mv2-sunset/) for planning your extension's migration.

> | Timeframe    | Microsoft Partner Center changes | Microsoft Edge changes |

Reviewer: ```suggestion
| Timeframe | Microsoft Partner Center changes | Microsoft Edge changes |
```

|--- |--- |--- |

> | July 2022 (already in effect)    | Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |

Reviewer: ```suggestion
| July 2022 (already in effect) | Microsoft Partner Center will no longer accept new Manifest V2 extensions with visibility set as `Hidden` or `Public`. | No change. |
```

| TBD - See note above | Microsoft Partner Center will no longer accept updates to existing Manifest V2 extensions. Developers can submit updates for migrating a V2 extension to V3. | Microsoft Edge stops running Manifest V2 extensions. Enterprises can allow Manifest V2 extensions to run on Microsoft Edge using Enterprise policies. |

> | TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policy. |

Reviewer: ```suggestion
| TBD - See note above | No change. | Manifest V2 extensions will no longer function in Microsoft Edge even with the use of Enterprise policies. |
```

> MV2 extensions will continue to be supported through the enterprise policy at least until Chromium MV2 support timeline (which is currently January 2024). 

Reviewer: ```suggestion
MV2 extensions will continue to be supported through the enterprise policy at least until Chromium Manifest V2 support timeline (which is currently January 2024). 
```

> Manifest V2 extensions will continue to be supported through the enterprise policy at least until Chromium Manifest V2 support timeline (which is currently January 2024). 

Reviewer: ```suggestion
Manifest V2 extensions will continue to be supported through Enterprise policies at least until Chromium Manifest V2 support timeline (which is currently January 2024). 
```
